libraries/AP_GPS/AP_GPS_SBF.h

```diff
@@ -66,6 +66,18 @@ class AP_GPS_SBF : public AP_GPS_Backend
     "sem, PVT, 5\n",
     "spm, Rover, all\n",
     "sso, Stream2, Dsk1, postprocess+event+comment+ReceiverStatus, msec100\n"};
+    const char* _initialisation_blob_i[5] = {
+    "sso, Stream1, COM1, PVTGeodetic+INSNavGeod+DOP+ReceiverStatus+VelCovGeodetic, msec100\n",
+    "srd, Moderate, UAV\n",
+    "sem, PVT, 5\n",
+    "spm, Rover, all\n",
+    "sso, Stream2, Dsk1, postprocess+event+comment+ReceiverStatus, msec100\n"};
+    const char* _initialisation_blob_dualantenna[5] = {
+    "sso, Stream1, COM1, PVTGeodetic+AttEuler+DOP+ReceiverStatus+VelCovGeodetic, msec100\n",
```

Comment:
You need to grab AttCovEuler as well to get the heading accuracy number. This is going to be playing an increasingly important role in the EKF handling.

Reply:
`    // yaw in degrees if available
    bool gps_yaw_deg(uint8_t instance, float &yaw_deg, float &accuracy_deg) const {
        if (!have_gps_yaw(instance)) {
            return false;
        }
        yaw_deg = state[instance].gps_yaw;
        // None of the GPS backends can provide this yet, so we hard
        // code a fixed value of 10 degrees, which seems like a
        // reasonable guess. Once a backend can provide a proper
        // estimate we can implement it
        accuracy_deg = 10;
        return true;
    }`

I assume it is for the accuracy_deg variable, that is now hard coded on 10 ?

Reply:
This is located in AP_GPS.h

Reply:
line 274

Reply:
Argh. I got confused. I have a patch which actually allows us to log the yaw accuracy and pass it to the EKF rather then the hard coded value. It's a pretty small patch though so I might get it in later this week. Anyways until I've done that we can ignore this comment, because the data you would be filling in isn't available yet.

Reply:
Ok, I already added the AttCovEuler message and parsed the Cov_HeadHead as the gps_yaw_accuracy (I did this before i read tour latest comment). The only thing is that the result is in degree².  I added this so it can be used in your your patch.